books/free-programming-books-langs.md

```diff
@@ -2256,6 +2257,7 @@ Books on general-purpose programming that don't focus on a specific language are
 * [Kestrels, Quirky Birds, and Hopeless Egocentricity](https://leanpub.com/combinators/read) - Reg Braithwaite
 * [Learn Ruby on Rails: Book One](https://learn-rails.com/content/learnrails1) - Daniel Kehoe
 * [Learn Ruby on Rails: Book Two](https://learn-rails.com/content/learnrails2) - Daniel Kehoe
+* [N+1 Queries Basics](https://bhserna.com/n-plus-1-queries-basics.html) - Benito Serna
```

Comment:
in general, we don't list books that require email addresses

Reply:
Thank @eshellman. Noted